redfish.conf

```diff
@@ -15,6 +15,10 @@ api_version = 300
 userName = administrator
 password =
 
+[event_service]
```

Comment:
Should we add a new section on README????

Reply:
Done!